File: IMS/src/app/features/customer/dtos/customerProfileResponse.dto.ts
export interface CustomerProfileResponseDto {
    name: string;
    username: string;
    customerId:string;
    email: string;
    phone: string;
    address: string;
    userId: string; // GUIDs are typically represented as strings in TypeScript
    dateOfBirth: string; // DateTime objects are usually represented as ISO 8601 strings
  }

File: IMSIntrim/Applications/DTOs/CustomerProfileResponseDto.cs
namespace IMSIntrim.Applications.DTOs
{
    public class CustomerProfileResponseDto
    {
        public required string Name { get; set; }
       
        public required string Email { get; set; }
        public required string Phone { get; set; }
        public  required string Address { get; set; } 
        public required int CustomerId { get; set; }
        public required string Username { get; set; }
        public Guid UserId { get; set; }
        public DateTime DateOfBirth { get; set; }
    }
}
